src/utils/is-touch.ts

```diff
@@ -0,0 +1 @@
+export const isTouch: boolean = typeof window === 'object' && 'orientation' in window;
```

Comment:
this folder has index and reexport, do this plz
https://github.com/wix/stylable-components/blob/4b2651b0bb14d0d4c658835baf2f3b97f502f9c1/src/utils/index.ts

and use it then from `../../utils`